api/maven-api-core/src/main/java/org/apache/maven/api/services/ProjectBuilderResult.java

```diff
@@ -70,7 +70,7 @@
      * @return The problems that were encountered during the project building, can be empty but never {@code null}.
      */
     @Nonnull
-    Collection<ProjectBuilderProblem> getProblems();
+    Collection<BuilderProblem> getProblems();
```

Comment:
Stupid question: Is it really a problem with the builder or a problem with the build?